components/authz-service/storage/postgres/policy.go

```diff
@@ -140,6 +140,26 @@ func (p *pg) ListPolicies(ctx context.Context) ([]*storage.Policy, error) {
 	return pols, nil
 }
 
+func (p *pg) GetUserPolicies(ctx context.Context, memberType, username, connectorId string) ([]string, error) {
+	rows, err := p.db.Query(`SELECT iam_policies.id FROM iam_members
```

Comment:
This query fetches the user policies for a given user/team. can it be extended to accept the list instead of one value, so that all the policies related to user and team can be fetched in one iteration instead of multiple calls?

Reply:
yes, i've updated the code.